Specifications/TalkToFeelbot.feature

```diff
@@ -0,0 +1,6 @@
+Scenario : TalkToFeelBotOnMobileOrLaptop
+I want to be able to talk to FeelBot on my laptop
+
+Given : That FeelBot is able to maintain a conversation
```

Comment:
Do you mean that if I am talking on a device and switch to another one, the conversation with the bot is still on ?